Dockerfile

```diff
@@ -28,6 +28,8 @@ RUN cd /src && npm install --production --unsafe-perm
 
 ADD . /src
 
+RUN cd /src/modules/hublin-easyrtc-connector && npm install
```

Comment:
can you change to `npm install --production` to avoid installing development dependencies?